pkg/controller/grouping/group_entity_index.go

```diff
@@ -672,7 +672,7 @@ func (i *GroupEntityIndex) Run(stopCh <-chan struct{}) {
 			klog.Info("Stopping GroupEntityIndex")
 			return
 		case group := <-i.eventChan:
-			parts := strings.Split(group, "/")
+			parts := strings.SplitN(group, "/", 2)
```

Comment:
Yes it's a critical change, since we use "namespace/name" for Group, and "name" for ClusterGroup, if we do not limit the split, this will only get the "namespace" for Group.